fleetdm/fleet
# fix: replace Zoom FMA with Zoom for IT

> Related issue: #23686

# Checklist for submitter

If some of the following don't apply, delete the relevant line.



  See [Changes files](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Committing-Changes.md#changes-files) for more information.
- [x] If database migrations are included, checked table schema to confirm autoupdate
- For database migrations:
  - [x] Checked schema for all modified table for columns that will auto-update timestamps during migration.
  - [x] Confirmed that updating the timestamps is acceptable, and will not cause unwanted side effects.
  - [x] Ensured the correct collation is explicitly set for character columns (`COLLATE utf8mb4_unicode_ci`).

---

## QA steps

1.  Start up Fleet. Check the DB `cron_stats` table and the `fleet_library_apps` table to make sure that the FMA cron job was scheduled and ran successfully
2. Apply the migrations and re-start Fleet
3. Check that 
  a. The previously run FMA job(s) in the `cron_stats` table have been deleted
  b. There should now be just 1 scheduled job in the `cron_stats` table, which should have been created ~30s after you re-started Fleet after the migration
  c. The FMA for Zoom should be replaced by "Zoom for IT" in the list of FMAs.
4. Add the new Zoom FMA to a setup experience for some team. Run the setup experience and validate Zoom was installed, but that no windows pop up during setup experience.
5. Uninstall the FMA, validate that uninstall is successful
